how-to/how-to-use-kata-containers-with-acrn.md

```diff
@@ -26,7 +26,29 @@ Please refer to ACRN [documentation](https://projectacrn.github.io/latest/index.
 This document requires the presence of the ACRN hypervisor and Kata Containers on your system. Install using the instructions available through the following links:
 
 - ACRN supported [Hardware](https://projectacrn.github.io/latest/hardware.html#supported-hardware).
-- ACRN [software](https://projectacrn.github.io/latest/getting-started/apl-nuc.html#use-the-script-to-set-up-acrn-automatically) setup.
+  > **Note:** Please make sure to have a minimum of 4 logical processors (HT) or cores.
+- ACRN [software](https://projectacrn.github.io/latest/tutorials/kbl-nuc-sdc.html#use-the-script-to-set-up-acrn-automatically) setup.
+- For networking, ACRN supports either MACVTAP or TAP. If MACVTAP is not enabled in the Service OS, please follow the below steps to update the kernel:
+
+  ```bash
```

Comment:
Could you remove the `bash` suffix from these two code blocks (or specify ` ```sh `) as this is only pseudo-shell, and not designed for users (or scripts) to execute directly.

Reply:
yes used ```sh